test/Interop/Cxx/namespace/class-inline-namespace-irgen.swift

```diff
@@ -0,0 +1,100 @@
+// RUN: rm -rf %t
+// RUN: split-file %s %t
+// RUN: %target-swift-emit-ir -I %t/Inputs -enable-cxx-interop %t/test.swift | %FileCheck %t/test.swift
+
+
+//--- Inputs/module.modulemap
+module std {
+  header "string.h"
+  requires cplusplus
+}
+//--- Inputs/string.h
+namespace std { inline namespace __1 {
```

Comment:
Also, let's not name this `std`. 

A more general comment: can you update all the names in this test to describe why they're interesting? For example, something like this:
```
namespace Parent {
struct InParent {};
namespace InlineChild {
struct InInlineChild {};
namespace SecondInlineChild {
struct InSecondInlineChild {};
} // namespace SecondInlineChild
} // namespace InlineChild
} // namespace Parent
```